charts/dvpe-deployment-gloo/values.yaml

```diff
@@ -361,14 +361,14 @@ istio:
 # Certificate value section            #
 # -------------------------------------#
 certificate:
-      # certificate.useCustomIssuer -- If set to `true` the default issuer's configuration settings will be used. This requires a custom issuer to be installed.
-      useCustomIssuer: true
-      # certificate.customIssuerSel -- custom issuer selector
+      # certificate.customIssuer -- predefined custom issuer selector; allowed values: internet, intranet
+      customIssuer: intranet
+      # certificate.customIssuerSel -- custom issuer selector; ignored if `customIssuer` is set
       customIssuerSel:
```

Comment:
I would rename that to customIssuerSelector since Sel could be anything and Selector is more precisely explaining what this property is meant for